Xamarin.Forms.Platform.UAP/Forms.cs

```diff
@@ -40,18 +40,22 @@ public static void Init(IActivatedEventArgs launchActivatedEventArgs, IEnumerabl
 			Device.SetFlags(s_flags);
 			Device.Info = new WindowsDeviceInfo();
 
-			switch (DetectPlatform())
+			switch (Windows.System.Profile.AnalyticsInfo.VersionInfo.DeviceFamily)
 			{
-				case Windows.Foundation.Metadata.Platform.Windows:
+				case "Windows.Desktop":
 					Device.SetIdiom(TargetIdiom.Desktop);
```

Comment:
I think we need to separate the `Surface` tablet from the desktop
```suggestion
					if (Windows.UI.ViewManagement.UIViewSettings.GetForCurrentView().UserInteractionMode ==
						Windows.UI.ViewManagement.UserInteractionMode.Touch)
						Device.SetIdiom(TargetIdiom.Tablet);
					else
						Device.SetIdiom(TargetIdiom.Desktop);
```

Reply:
does that UserInteractionMode  work only when you Switch to tablet mode on touch capable devices? Because if not Surface but any other Notebook with touch Screen will fall into this category also.

Reply:
In fact, it will depend on the system settings. [Which can be changed at any time.](https://docs.microsoft.com/en-us/uwp/api/windows.ui.viewmanagement.userinteractionmode#remarks)

> Some devices (PC, laptop, tablet) support both a Desktop (mouse-optimized) and a Tablet (touch-optimized) mode.
Users can switch between running in Tablet mode and Desktop mode by going to Settings > System > Tablet mode and setting Make Windows more touch-friendly when using your device as a tablet.

For tablets the default will be `Touch` mode, for desktops `Mouse` mode. 
Touch mode indicates that the user has tablet mode enabled.
![image](https://user-images.githubusercontent.com/27482193/54285800-afa80f00-45b3-11e9-8adc-03b9bc3ea261.png)